nodeup/pkg/model/nerdctl.go

```diff
@@ -0,0 +1,65 @@
+package model
+
+import (
+	"fmt"
+
+	"k8s.io/klog/v2"
+	"k8s.io/kops/upup/pkg/fi"
+	"k8s.io/kops/upup/pkg/fi/nodeup/nodetasks"
+	"k8s.io/kops/util/pkg/distributions"
+)
+
+type NerdctlBuilder struct {
+	*NodeupModelContext
+}
+
+var _ fi.NodeupModelBuilder = &NerdctlBuilder{}
+
+func (b *NerdctlBuilder) Build(c *fi.NodeupModelBuilderContext) error {
+	if b.skipInstall() {
+		klog.Info("SkipInstall is set to true; won't install nerdctl")
```

Comment:
Nit: Maybe `containerd.skipInstall is set to true; won't install nerdctl` or maybe `not installing containerd; won't install nerdctl`